eng/pipelines/common/platform-matrix.yml

```
@@ -84,9 +87,14 @@ jobs:
       archType: x64
       platform: Linux_musl_x64
       container:
-        image: alpine-3.9-WithNode-0fc54a3-20190918214015
+        # alpine coreclr cmake errors on newer builds
+        ${{ if eq(parameters.runtimeFlavor, 'mono') }}:
+          image: alpine-3.9-WithNode-0fc54a3-20200121145958
+        ${{ if ne(parameters.runtimeFlavor, 'mono') }}:
```

Can we open an issue for this and investigate? We should try to be consistent with the docker images we build with. 

Also, instead of using `!= mono` I would change this to `== coreclr` so that it is easier to read.

```suggestion
        ${{ if eq(parameters.runtimeFlavor, 'coreclr') }}:
```

---

Yes I'll file an issue for that as it's highly concerning. Just hadn't had time yet.